scripts/localVsDissecLearning.js

```diff
@@ -48,31 +53,21 @@ const runExperiment = async () => {
   })
 
   // Download all bank operations
-  const allOperations = await client.queryAll(Q(BANK_DOCTYPE))
+  const sortedOperations = await client.queryAll(
+    Q(BANK_DOCTYPE).where({}).sortBy([{ date: 'asc' }])
```

Comment:
I would suggest the following: 
```
Q(BANK_DOCTYPE)
  .where({date: { $gt: null}})
  .sortBy([{ date: 'asc' }])
  .indexFields(['date'])
```
This way, you ensure an index on the date fields (theoretically, it does the same thing thanks to cozy-client, but I think it's better to be explicit)